content/yaml/yaml-getting-started.md

```diff
@@ -131,6 +131,9 @@ workflows:
           - name@example.com
       scripts:
         - echo 'Post-publish script'
+      labels:
```

Comment:
I'm a bit confused. Are labels part of `publishing` or are they defined at the workflow level (as in the example below) or are both supported?

Reply:
Updated [PR](https://github.com/codemagic-ci-cd/codemagic-docs/pull/913/commits/c90e4253b75e66cd931ef11d3a0f16c2df514386)

Reply:
`labels` are still under `publishing` and not directly under `workflow` here.